File: jbake-core/src/main/java/org/jbake/util/DebugUtil.java
package org.jbake.util;

import java.io.PrintStream;
import java.util.Map;

public class DebugUtil {
    public static <T extends Object> void printMap(Map<String, T> map, PrintStream printStream) {
        printStream.println();
        for (Map.Entry<String, T> entry: map.entrySet()) {
            printStream.println(entry.getKey() + " :: " + entry.getValue());
        }
        printStream.println();
    }
}

File: jbake-core/src/main/java/org/jbake/util/DebugUtil.kt
package org.jbake.util

import java.io.PrintStream

object DebugUtil {
    fun <T : Any?> printMap(map: MutableMap<String?, T?>, printStream: PrintStream) {
        printStream.println()
        for (entry in map.entries) {
            printStream.println(entry.key + " :: " + entry.value)
        }
        printStream.println()
    }
}
